README.md

```diff
@@ -48,68 +45,192 @@ $ ./gradlew test
+by adding the `-javaagent` parameter to your JVM properties and enable JFR monitoring using one of the following mechanisms:
+
+Agent yaml config (the following should be nested under the `common` stanza):
+
+```yaml
+  # Real-time profiling using Java Flight Recorder (JFR).
+  # This feature reports dimensional metrics to the ingest endpoint configured by
+  # metric_ingest_uri and events to the ingest endpoint configured by event_ingest_uri.
+  # Both ingest endpoints default to US production but they will be automatically set to EU
+  # production when using an EU license key. Other ingest endpoints can be configured manually.
+  # Requires a JVM that provides the JFR library.
+  jfr:
+
+    # Set to true to enable Real-time profiling with JFR.
+    # Default is false.
+    enabled: true
+
+    # Set to true to enable audit logging which will display all JFR metrics and events in each harvest batch.
+    # Audit logging is extremely verbose and should only be used for troubleshooting purposes.
+    # Default is false.
+    audit_logging: false
 ```
-<dependency>
-    <groupId>com.newrelic</groupId>
-    <artifactId>jfr-mappers</artifactId>
-    <version>1.2.0</version>
-</dependency>
+
+System property:
+
+```
+-Dnewrelic.config.jfr.enabled=true
 ```
 
-#### gradle dependency
+Environment variable:
 
 ```
-compile group: 'com.newrelic', name: 'jfr-mappers', version: '1.2.0'
+export NEW_RELIC_JFR_ENABLED=true
+```
+
+By default, the New Relic Java agent will send JFR data to New Relic US production metric and event ingest endpoints.
+JFR data will be associated with the APM entity generated by the New Relic Java agent as defined by your `app_name` and `license_key`. 
+To change this or other behavior see [configuration for New Relic Java agent JFR service](#configuration-for-new-relic-java-agent-jfr-service).
+
+#### Configuration for New Relic Java agent JFR service
+
+When using the New Relic Java agent, the JFR service fully respects agent configuration settings.
+
+The New Relic Java agent is configurable via a
+[yaml configuration file](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/),
+[system properties](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#System_Properties),
+and [environment variables](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#Environment_Variables).
+
+The following agent configuration options affect how and where JFR data is reported:
+* [app_name](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#cfg-app_name)
+* [license_key](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#cfg-license_key)
+* [jfr.enabled]()
+* [jfr.audit_logging]()
+* [metric_ingest_uri]()
+* [event_ingest_uri]()
```

Comment:
FYI these links are published yet. I'm planning on updating them when a batch of docs goes live.